pkg/cloud/services/ec2/vpc.go

```diff
@@ -189,26 +192,10 @@ func (s *Service) createVPC() (*infrav1.VPCSpec, error) {
 		return nil, errors.Wrapf(err, "failed to wait for vpc %q", *out.Vpc.VpcId)
 	}
 
-	// Apply tags so that we know this is a managed VPC.
-	tagParams := s.getVPCTagParams(*out.Vpc.VpcId)
-	if err := wait.WaitForWithRetryable(wait.NewBackoff(), func() (bool, error) {
-		if err := tags.Apply(&tags.ApplyParams{
-			EC2Client:   s.scope.EC2,
-			BuildParams: tagParams,
-		}); err != nil {
-			return false, err
-		}
-		return true, nil
-	}, awserrors.VPCNotFound); err != nil {
-		record.Warnf(s.scope.AWSCluster, "FailedTagVPC", "Failed to tag managed VPC %q: %v", *out.Vpc.VpcId, err)
-		return nil, err
-	}
-	record.Eventf(s.scope.AWSCluster, "SuccessfulTagVPC", "Tagged managed VPC %q", *out.Vpc.VpcId)
-
 	return &infrav1.VPCSpec{
 		ID:        *out.Vpc.VpcId,
 		CidrBlock: *out.Vpc.CidrBlock,
-		Tags:      infrav1.Build(tagParams),
+		Tags:      infrav1.Build(s.getVPCTagParams(*out.Vpc.VpcId)),
```

Comment:
Is it not possible to use the output of the call to `s.scope.EC2.CreateVpc()` here?

Reply:
Probably. I'll add a ec2 tags to infrav1 tags conversion func.